src/components/o-modal/index.scss

```diff
@@ -48,19 +46,14 @@ $close-button-color-hovered: #000;
 
 .m-modal__close-button {
   float: right;
-  color: $close-button-color;
+  color: $color-axa-blue;
 }
 
 .m-modal__close-button:hover,
```

Comment:
In case you need to rebuild a button, you have plenty of generic mixins available from:
https://github.com/axa-ch/patterns-library/blob/develop/src/components/m-button/mixins/_button.scss

Reply:
Hi Andy, this button is more or less only an "X" to close the modal. But good point anyways, I would say as soon as we need that X somewhere else too, I could move it to this button sass file.

Reply:
Ah I see, then you can ignore my comment